crossplane.yaml

```diff
@@ -3,6 +3,52 @@ kind: Configuration
+
+    friendly-name.meta.crossplane.io: AWS Reference Platform
+    friendly-group-name.meta.crossplane.io/aws.platformref.crossplane.io: AWS
+
+    friendly-kind-name.meta.crossplane.io/compositenetwork.aws.platformref.crossplane.io: Composite Network
+    friendly-kind-name.meta.crossplane.io/network.aws.platformref.crossplane.io: Network
+
+    friendly-kind-name.meta.crossplane.io/compositecluster.aws.platformref.crossplane.io: Composite Kubernetes Cluster
+    friendly-kind-name.meta.crossplane.io/cluster.aws.platformref.crossplane.io: Kubernetes Cluster
+
+    friendly-kind-name.meta.crossplane.io/services.aws.platformref.crossplane.io: Cluster Services
```

Comment:
another thing i've been thinking about (not in scope for this PR but just on my mind) is that these services and EKS types are not ever really intended to be created directly - they are composite types nested within parent composite types.  i wonder if we should in the future think about how to expose these for scenarios like understanding the full composition hierarchy during debugging, but not exposing them as something that can be created directly by the user.